validate/__main__.py

```diff
@@ -24,39 +24,40 @@
 def main():
     '''Main function and CLI tool'''
     args = parse_args(sys.argv[1:])
-    path = Path(args.path)
+    paths = [Path(i) for i in args.path]
     input_type = args.type
     file_type = input_type
 
-    try:
-        path_exists(path)
-    except IOError as err:
-        sys.exit(f"Error: {args.path} {str(err)}")
-
-    try:
-        if input_type in FILE_TYPES_DICT:
-            validate_file(path, input_type)
-        elif input_type == GENERIC_FILE_TYPE:
-            file_type, _ = detect_file_type_and_ext(path)
-            validate_file(path, file_type)
-        elif input_type in DIR_TYPES:
-            validate_dir(path, input_type)
-        elif input_type in CHECKSUM_GEN_TYPES:
-            create_checksum_file(path, input_type)
-    except TypeError as err:
-        sys.exit(f"Error: {args.path} {str(err)}") # raise errors, has implicit exit code
-    except ValueError as err:
-        sys.exit(f"Error: {args.path} {str(err)}")
-    except (IOError, OSError) as err:
-        sys.exit(f"Error: {args.path} {str(err)}")
-
-    print(f"Input: {args.path} is valid {file_type}")
+    for path in paths:
```

Comment:
Consider moving the list comprehension here: `for path in [Path(pathname) for pathname in args.path]:`